lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```

Comment:
Maybe we should make this a flag based on "themeType: dark/light" since some people might want a skin?